packages/react/src/TreeView/TreeView.tsx

```diff
@@ -282,6 +289,7 @@ const Root: React.FC<TreeViewProps> = ({
   'aria-labelledby': ariaLabelledby,
   children,
   flat,
+  truncate = true,
```

Comment:
@TylerJDev do you think it might make more sense to make this false by default? Just wondering if this is kind of a breaking change. Have you run an integration test?

Reply:
For this, I set it to `true` to keep the default behavior of the text truncating when not using the prop. When using it with `false`, it will wrap instead. The new behavior should only be opt-in, so that we can release this under a non breaking change. 

Let me know if you think we should change the prop name (e.g. `shouldNotTruncate`, `wrap`). Not sure if `truncate` is confusing or not 😅